subsys/bluetooth/controller/ll_sw/ll_filter.h

```diff
@@ -13,11 +13,12 @@ struct ll_filter {
 };
 
 void ll_filter_reset(bool init);
+void ll_filters_update(int adv_fp, int scan_fp);
```

Comment:
Well, logically this fits better together: a single function to update the filters whenever activity starts. I don't mind splitting this up though, I'll take a look